target/linux/ipq40xx/base-files/etc/board.d/03_gpio_switches

```diff
@@ -27,6 +27,9 @@ mikrotik,hap-ac3)
 zte,mf286d)
 	ucidef_add_gpio_switch "power_btn_block" "Power button blocker" "421" "0"
 	;;
+zte,mf289f)
```

Comment:
This can be combined with the above, like this:
```
zte,mf286d|\
zte,mf289f)
```
Also, please double-check the GPIO number.